src/shared/utils.ts

```diff
@@ -35,12 +35,15 @@ export function findNetStandardTargetFramework(tfmShortNames: string[]): string
     return tfmShortNames.find((tf) => tf.startsWith('netstandard'));
 }
 
-export function isWebProject(projectPath: string): boolean {
+export function isWebProject(projectPath: string): [boolean, boolean] {
     const projectFileText = fs.readFileSync(projectPath, 'utf8');
 
     // Assume that this is an MSBuild project. In that case, look for the 'Sdk="Microsoft.NET.Sdk.Web"' attribute.
     // TODO: Have OmniSharp provide the list of SDKs used by a project and check that list instead.
-    return projectFileText.toLowerCase().indexOf('sdk="microsoft.net.sdk.web"') >= 0;
+    return [
+        projectFileText.toLowerCase().indexOf('sdk="microsoft.net.sdk.web"') >= 0,
```

Comment:
nit: Save `projectFileText.toLowerCase()` into a variable so it only needs to be done once.

Reply:
done.